debian/appscale_build.sh

```diff
@@ -1,7 +1,31 @@
 #!/bin/bash
+#
+# This script install appscale on the local machine. It pulls in all the
+# needed dependencies and configure them properly for AppScale.
+#
+
+# Some basic check: we need a way to install packages.
+PKG_CMD="$(which apt-get)"
+if [ -z "${PKG_CMD}" ]; then
+    echo "Cannot find the package manager command!"
+    exit 1
+fi
 
 set -e
 
+# Update the pakcages list and cache.
+echo -n "Updating package list and cache ..."
+${PKG_CMD} update > /dev/null
+echo "done."
+
+# We need to make sure we have lsb-release, before we use it. On
+# streamlined images (like docker) it may not be present.
+if ! which lsb_release > /dev/null ; then 
+    echo -n "Installing lsb-release..."
+    (${PKG_CMD} install -y lsb-release) > /dev/null
```

Comment:
not really, it seems cute at the time :) I had more commands there, so I used it for the redirection: the refactor brought it down to a single command: I will remove the subshell. Thanks!